cosmwasm/contract.ts

```diff
@@ -0,0 +1,433 @@
+import { StdFee } from '@cosmjs/stargate';
+import { Command, Option } from 'commander';
+
```

Comment:
upload/instantiate/execute should be added here in a follow up PR as well since they can be run via an EOA or governance (you can create it parallelly)